# Day 1 Homework Python Basics

Question 1

In [1]:
fname = input("enter your first name")
lname = input("enter your last name")
print(lname + " " + fname)

enter your first nameChris
enter your last nameCabrera
Cabrera Chris


Question 2

In [4]:
x = int(input("enter a number"))
xx = (x*11)
xxx = (x*111)
ans = x + xx + xxx
print(ans)

enter a number5
615


Question 3

In [1]:
country = input("What country are you from?")
print("I have heard that "+ country +" is a beautiful country!")

What country are you from?Peru
I have heard that Peru is a beautiful country!


Question 4

In [2]:
x = 10
y = 50
if (x ** 2 > 100 and y < 100):
    print(x, y)

no output since x^2 is 100, and 100 is not greater than 100


Question 5

In [4]:
a = [10, 20]
b = a
b += [30, 40]
print(a)
print(b)

[10, 20, 30, 40]
[10, 20, 30, 40]


[10, 20, 30, 40]
[10, 20, 30, 40]
This is due to the "pointer variable" property


Question 6

In [6]:
print(2%6)

2


It is 2 because if you divide 2 by 6 your remainder will be 2


Question 7

In [8]:
print(2 * 3 ** 3 * 4)

216


This simplifies to 2 * (3^3)*4 or 8 *27 which equals 216


Question 8

a text editor is a program that allows you to type text which can then be interpreted into a specific format


Question 9

Python is a language that was originally developed as a training language, but has been dveloped into a full fledged programming language


Question 10

Jupyter is an application that allows you to combine code sections with text sections in a notebook like fashion. Two alternatives are PyCharm and Appache Zeppelin.